packages/component-submission/client/utils/getSuggestedTitle.js

```diff
@@ -0,0 +1,22 @@
+import { sortBy, last } from 'lodash'
```

Comment:
Could we make this `getSuggestion(fieldName)` that way we could get any suggestion out of this?

Reply:
done